appcenter/AppCenter.d.ts

```diff
@@ -0,0 +1,21 @@
+export const enum AppCenterLogLevel {
+    LogLevelVerbose = 2,
+    LogLevelDebug = 3,
+    LogLevelInfo = 4,
+    LogLevelWarning = 5,
+    LogLevelError = 6,
+    LogLevelAssert = 7,
+    LogLevelNone = 99
+}
+
+export interface CustomProperties {
+    set(key: string, value: string);
```

Comment:
Please look at our docs and also Android SDK, we have multiple types for this one.

Reply:
Looks like a simple map on both, let me know if this works.